### PR TITLE
Python 3 support

### LINKED ISSUE #53: python3 support

### PR DESCRIPTION
**Pull request by eldond**
Description:
- Close #53 
- Move tests to python3

Pre-merge checklist (in addition to automatic checks):
- [x] Author has reviewed and finalized code
- [x] New functions, classes, and keywords are described in docstrings
- [x] This pull request is ready for review / merge

### CODE REVIEW

File: pgmpl/figure.py [hunk 1]
```diff
@@ -196,10 +201,14 @@ def gca(self):
         Imitation of matplotlib gca()
         :return: Current axes for this figure, creating them if necessary
         """
+        if self.axes is not None:
+            try:
+                ax = list(flatten(np.atleast_1d(self.axes)))[-1]
+            except RuntimeError:  # Happens if Qt has deleted the Axes
+                print('Warning: Qt has deleted the axes; figure had a residual reference to a bad Qt object. (gca)')
+                self.axes = None
```
Comment (eldond) [inline pgmpl/figure.py:209]: I think codecov is complaining about these three lines not executing in the test. Ugh. This is a weird error to begin with, and it IS being hit earlier (lines 140-142) where I have to try/except to catch the same thing. I'm going to just accept this for now. Maybe I can figure out what exactly causes this and how to reliably reproduce it to hit these three, too. As it is, I can't even hit these errors by running tests individually; it's some combination that does this, I guess. 

So, proceeding against codecov complaint.

**Comment by eldond** [comment]
Not clear why travis isn't posting anymore, but it is passing. So... here we go.

**Comment by eldond** [comment]
@orso82 This thing's ready for python 3. You can use it!

**Comment by orso82** [comment]
Great @eldond ! mention @FantuzziMatteo97

**Pull request merged**